keyboards/cannonkeys/satisfaction75/led.c

```diff
@@ -80,6 +80,21 @@ void backlight_init_ports(void) {
   }
 }
 
+void suspend_power_down_user(void) {
+    backlight_set(0);
+}
+void suspend_wakeup_init_user(void) {
+  if(kb_backlight_config.enable){
+    if(kb_backlight_config.breathing){
```

Comment:
If suspend messes up backlight breathing then that is a bug in the backlight feature and should be fixed there instead.

Reply:
I am hardly an expert, I am just going by what I've gleamed from looking over the led.c, but I believe the backlight and backlight breathing is implemented separate from the base qmk code in order for it to be configured from the OLED? I could be totally wrong. My change just sets the backlight to 0 on suspend, and then on wakeup restores the previous setting by executing part of backlight_init_ports.

Reply:
Ah, this board has a custom backlight implementation. I'm not sure if that's necessary though...

Reply:
Yeah, possibly, but that's beyond me.